pkg/processor/processor.go

```diff
@@ -63,24 +63,52 @@ func (p *Processor) run() {
 // For instance, we want to add the timestamp, hostname and a log level
 // to messages coming from a file
 func (p *Processor) computeExtraContent(msg message.Message) []byte {
+	// if the first char is '<', we can assume it's already formatted as RFC5424, thus skip this step
+	// (for instance, using tcp forwarding. We don't want to override the hostname & co)
 	if len(msg.Content()) > 0 && msg.Content()[0] != '<' {
 		// fit RFC5424
 		// <%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %$!new-appname% - - - %msg%\n
-		timestamp := time.Now().UTC().Format("2006-01-02T15:04:05.000000+00:00")
-		extraContent := []byte("<46>0 ")
-		extraContent = append(extraContent, []byte(timestamp)...)
+		extraContent := []byte("")
+
+		// Severity
+		if msg.GetSeverity() != nil {
```

Comment:
Should the default be part of the msg?

Reply:
It could, but then I would ask the same question about the timestamp: should we default to `time.now` in the message construction?

For timestamp it would not be great, as we use the message's timestamp in the auditor (for docker for instance), and as a result the registry would look more confusing*.
As a result, I thought for consistency we'd keep the default severity out of message too. However I don't have a strong opinion on this.

* for instance, it would go from
```
{
    "Registry": {
        "file:/home/vagrant/logrotate/tail.log": {
            "LastUpdated": "2017-11-15T08:27:20.563981904Z",
            "Offset": 723842,
            "Timestamp": ""
        }
    },
    "Version": 1
}
```
to
```
{
    "Registry": {
        "file:/home/vagrant/logrotate/tail.log": {
            "LastUpdated": "2017-11-15T08:27:20.563981904Z",
            "Offset": 723842,
            "Timestamp": "2017-11-15T08:27:20.563981904Z"
        }
    },
    "Version": 1
}
```
and we wouldn't know which one is relevant between offset and timestamp without being aware of the logic